src/splitInteger.test.js

```diff
@@ -8,12 +8,22 @@ describe(`Function 'splitInteger':`, () => {
   });
 
   it(`should return an array`, () => {
+    expect(splitInteger(5, 1)).toBeInstanceOf(Array);
+  });
+
+  it(`should return an array [5] if (num = 5, parts = 1)`, () => {
+    expect(splitInteger(5, 1)).toEqual([5]);
+  });
```

Comment:
```suggestion
  it(`should return an array [5] if num = 5, parts = 1`, () => {
    expect(splitInteger(5, 1)).toEqual([5]);
  });
```

Reply:
Parentheses are not needed in the test title, it's should not look like JS code :)